bluebrain/deployment/environments/externals.yaml

```diff
@@ -74,8 +74,9 @@ spack:
     - clang-tools@12.0.1p2
     - clang-tools@11.1.0p2
     - nvhpc@21.11
-    - nvhpc@22.2
     - nvhpc@22.3
+    - nvhpc@22.5
```

Comment:
@olupton : could you remind me 22.5 was useful or just a buggy for us?

Reply:
IIRC we never deployed it because 22.7 was available before we resolved https://github.com/BlueBrain/CoreNeuron/issues/826.

I don't have a strong view -- I would mildly prefer to deploy it and then drop it once we have 22.9, as we have more confidence in 22.3 as an "old stable" release.